GraphClient.go

```diff
@@ -266,6 +271,30 @@ func (g *GraphClient) ListGroups(opts ...ListQueryOption) (Groups, error) {
 	return marsh.Groups, err
 }
 
+// ListUserGroups returns a list of all group ids the user is a member of
+// You can specify the securityGroupsEnabeled parameter to only return security group ids
+// Reference: https://docs.microsoft.com/en-us/graph/api/directoryobject-getmembergroups?view=graph-rest-1.0&tabs=http
+func (g *GraphClient) ListUserGroups(identifier string, securityGroupsEnabeled bool) ([]string, error) {
```

Comment:
I propose the following:
* to make this function private (beginning with lower case)
* raname to `getMemberGroups`
* add a parameter for `ctx context.Context`, since we want to support context in all our functions
* create wrapper-function in `User.go` and `Group.go` to call this function, named `GetMemberGroupsAsStrings` returning `([]string, error)`
* *Optional*: Create a wrapper-function in `User.go` and `Group.go` to call this function named `GetMemberGroups` returning `(Groups, error)` by recursively resolving all returned group IDs to their respective objects - that may be quite some API-calls, but then you can immediately use the object.

Please also write a unit test for this function.

Reply:
I don't understand the point of adding this func to `Group.go`. I've made it so it's an user method and it's called using his ID. I can make adjustments if necessary.

Reply:
Hi @Goorsky123 , it took a while, but now I have finally added some adjustments myself. Thank you for the great work reagrding this. I just wanted to make this function available for Groups as well - since it's supported by the msgraph API iteself.